### ISSUE TITLE
Utilizing powerful iPhone 13 Pro functionality with Seek

### ISSUE THREAD

**Issue by bcvilnrotter** [inaturalist/SeekReactNative#562]
Hi all, please let me know if this suggestion is inappropriate. I searched the issues tab for a similar issue but couldn't find one.

I'm currently using the Seek app on an iPhone 13 Pro, but notice I'm unable to unlock the full capabilities of my phones camera through the app. I wonder if it would be possible to allow the app to access my micro back camera, and other features that would really help me take great pictures of the species I'm trying to identify.

If this is not possible (as I have no idea the access iOS apps have with the camera software, or what kind of haul this will be) would alternatively there be any way to incorporate integration to link the iOS camera app to the Seek app? (either through Zapier / IFTTT / iOS shortcut automation / etc.)

Thanks all for your time.

**Comment by jtklein** [comment]
Yes, this is something we have on our list. I believe this https://github.com/inaturalist/SeekReactNative/issues/529 will enable the usage of different cameras of the phone.

**Comment by jtklein** [comment]
We have now released version 2.15 on iOS. This contains an update to the camera view in the app. Now it should give you a sensible default by showing the combination of different lenses available. So, e.g., if there is a macro lens on your phone and you zoom into an object close to the camera it should switch to the macro lens automatically for you.
